src/Elastic.Markdown/Slices/HtmlWriter.cs

```diff
@@ -81,7 +79,8 @@ private async Task<string> RenderLayout(MarkdownFile markdown, MarkdownDocument
 		var html = markdown.CreateHtml(document);
 		await DocumentationSet.Tree.Resolve(ctx);
 
-		var navigationHtml = await NavigationHtmlWriter.RenderNavigation(markdown.RootNavigation, ctx);
+		var isRoot = DocumentationSet.Tree.Id == markdown.RootNavigation.Id;
```

Comment:
Can this check live on this deleted line? 

https://github.com/elastic/docs-builder/pull/787/files#diff-d551730a87f5d4489d8ecb7870729ef7ac6bbec2b67e1fd2cdce4ee8a0243b17L52

The goal of `INavigationHtmlWriter` was to isolate the usage of `DocumentationSet.Tree` so that the assembler implementation of it does not have to rely on it (or an outside check for root).

Reply:
I figure I can also use `isPrimaryNavEnabled` https://github.com/elastic/docs-builder/pull/787/commits/502e04ac75840e52fa35355df47b4817e4622ec0

Will this also work for the assembler?

Reply:
In theory this will make the pages nav broken on the root page `/`. But since the landing page doesn't have a navigation it's ok.

Reply:
So clean! Yeah assembler will enable primary nav for all 👍